tests/test_bgp_neighbor.rb

```diff
@@ -26,86 +26,131 @@ class TestRouterBgpNeighbor < CiscoTestCase
   # rubocop:disable Style/ClassVars
   @@asn = 55
   @@addr = '1.1.1.1'
+  @@remote_asn = 99
   # rubocop:enable Style/ClassVars
```

Comment:
We can make Rubocop happy by changing these into class constants instead of class variables - probably the right thing to do too:

```ruby
ASN = 55
ADDR = '1.1.1.1'
REMOTE_ASN = 99
```